packages/usa-select/src/content/usa-select~multiple.json

```diff
@@ -0,0 +1,5 @@
+{
+  "disabled": false,
+  "aria_disabled": false,
+  "size": 0
+}
```

Comment:
You can set basic defaults and override a value or add a new prop easily.

Checkout banner settings for an example:
https://github.com/uswds/uswds/blob/8b78b05795cfa5d4998bfd19ae1977b1108edbe6/packages/usa-banner/src/usa-banner.stories.js#L21-L25

Reply:
Simplified story in c662f50928025ed1dc8c40e04b258e7f879cfb17. Now the `disabled/aria_disabled` settings are defaults and story uses a single template.